src/site/content/en/blog/index.njk

```diff
@@ -24,9 +28,61 @@ date: 2018-11-05
 
   <section class="w-grid">
     <div class="w-grid__columns w-grid__columns--three" role="list">
-      {% for post in collections.posts %}
+      {% for post in posts %}
         {% PostCard {post: post} %}
       {% endfor %}
     </div>
   </section>
+
+  <nav class="w-pagination">
+    {# Show first page anchor only if not on the first page #}
+    {% if page.url != pagination.firstPageHref %}
+      <div class="w-pagination__link">
+        <a aria-label="First Page" href="{{ pagination.firstPageHref | stripLanguage }}"><<</a>
+      </div>
+    {% endif %}
+
+    {# Show previous page anchor only if not on the first two pages #}
+    {% if pagination.pageNumber > 1 %}
+      <div class="w-pagination__link">
+        <a aria-label="Previous Page" href="{{ pagination.previousPageHref | stripLanguage }}"><</a>
+      </div>
+    {% endif %}
+
+    {%- for pageEntry in pagination.hrefs | paginate(page.url)  %}
+      <div
+        {% if page.url == pageEntry.href %}
+          class="w-pagination__active"
+        {% else %}
+          class="w-pagination__link"
+        {% endif %}
+      >
+        {% if pageEntry.showEllipses %}
+          <span class="w-pagination__spacer">...</span>
```

Comment:
Use ellipsis character `…` instead of three dots. It'll announce better for screen readers.